docs/root/start/install.rst

```diff
@@ -63,6 +63,28 @@ You can install Envoy on Mac OSX using the official brew repositories.
       $ brew update
       $ brew install envoy
 
+.. _start_install_kubernetes:
+
+Install Envoy on Kubernetes using Envoy Gateway
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can run Envoy as a Kubernetes Ingress Gateway by installing Envoy Gateway on your Kubernetes cluster.
+
+The following commands will:
+- Install the Gateway API CRDs and Envoy Gateway
+- Wait for Envoy Gateway to become available
+- Install the GatewayClass, Gateway, HTTPRoute and an example app
+
+.. tabs::
+
+   .. code-tab:: console
+
+      $ helm install eg oci://docker.io/envoyproxy/gateway-helm --version v0.0.0-latest -n envoy-gateway-system --create-namespace
```

Comment:
wondering if there is a `latest` (as opposed to v0.0.0-latest) as im concerned this will become stale